### DISCUSSION TITLE
Additional sections to add

### DISCUSSION THREAD

**Discussion by cyclingwithelephants** [cyclingwithelephants/devs-on-a-boat#10] [Ideas]
I'm writing down some ideas for additional sections we might want to add in the future, please feel free to add your own or discuss those already mentioned ones!

- A contribution page where we can invite others to contribute, and outline how that might work. I think it'd be great to find a way for old salts who aren't developers to be able to share their knowledge (even if it's just by e.g. email and turned into markdown by a developer). Additionally I'd like a way to publicly thank y'all who are contributing and making this possible, possibly as part of this page.
- I'd like to make example wiring diagrams for all the electronics on the boats, and ultimately some networking stuff once I've got examples of that up. I envisage this will be a longer term goal.

**Comment by bergie** [comment]
Maybe also section for relevant links? For example this site has great resources for dev work on a boat https://100r.co/site/working_offgrid_efficiently.html

**Reply by cyclingwithelephants** [reply]
I like this idea a lot (and the blog! thanks for sharing). I'm struggling to come up with a decent name for the section, I guess there's the obvious/boring "relevant links" but the best I can come up with is "external resources" or "others wisdom"... I'm putting it up as external resources for now but am keen to get feedback
